src/components/article/ArticleFullCard.jsx

```diff
@@ -1,17 +1,37 @@
+import { useState } from "react";
+import { patchArticle } from "../../utils/api";
 import { getDateString } from "../../utils/date";
 
-function ArticleFullCard({article}) {
+function ArticleFullCard({ article, setArticle }) {
+
+	const voteOnArticle = () => {
+		setArticle(currentArticle => {
+			return { ...currentArticle, votes: currentArticle.votes + 1 };
+		});
+		patchArticle(article.article_id, { inc_votes: 1 })
+			.then(data => {})
```

Comment:
If you're not using the data, you dont need this line, can just have the function and the catch